<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QMainWindow" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>老鼠管理</string>
  </property>
  <property name="windowIcon">
   <iconset resource="logo.qrc">
    <normaloff>:/image/mouse.jpeg</normaloff>:/image/mouse.jpeg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tw_main">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>751</width>
      <height>461</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="usesScrollButtons">
     <bool>true</bool>
    </property>
    <property name="documentMode">
     <bool>true</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab_mouse">
     <attribute name="title">
      <string>小鼠信息</string>
     </attribute>
     <widget class="QTableView" name="table_mouse">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>731</width>
        <height>381</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QPushButton" name="btn_rf_mouse">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>400</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>更新</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_edit">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>400</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>信息设置</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_message">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>410</y>
        <width>281</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;双击打开详细信息&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_search">
     <attribute name="title">
      <string>查询</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Search By:</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>651</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Name</string>
      </property>
      <widget class="QCheckBox" name="check_name">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>By Name</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="edit_name">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>20</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QComboBox" name="box_name">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>20</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
       <property name="currentText">
        <string>EXACTLY</string>
       </property>
       <item>
        <property name="text">
         <string>EXACTLY</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>STARTS WITH</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ENDS WITH</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>651</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Date</string>
      </property>
      <widget class="QDateTimeEdit" name="birth_min">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>20</y>
         <width>181</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QCheckBox" name="check_birth">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>81</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Birth</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="check_lastmeal">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>50</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Lastmeal</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>20</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>between</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>50</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>between</string>
       </property>
      </widget>
      <widget class="QDateTimeEdit" name="birth_max">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>453</x>
         <y>20</y>
         <width>191</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateTimeEdit" name="lastmeal_min">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>50</y>
         <width>181</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateTimeEdit" name="lastmeal_max">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>453</x>
         <y>50</y>
         <width>191</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>651</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Health</string>
      </property>
      <widget class="QCheckBox" name="check_health">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>By Health</string>
       </property>
      </widget>
      <widget class="QComboBox" name="box_health">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>20</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>健康</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>良好</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>生病</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>极差</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QPushButton" name="btn_search">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>651</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_reset">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>330</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>651</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Master</string>
      </property>
      <widget class="QCheckBox" name="check_master">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>By Master</string>
       </property>
      </widget>
      <widget class="QComboBox" name="box_master">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>20</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_mousedeath">
     <attribute name="title">
      <string>小鼠死亡列表</string>
     </attribute>
     <widget class="QTableView" name="table_mouse_death">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>741</width>
        <height>381</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectItems</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>400</y>
        <width>721</width>
        <height>27</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btn_cm_death">
         <property name="text">
          <string>提交</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_rb_death">
         <property name="text">
          <string>撤销</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_ad_death">
         <property name="text">
          <string>添加</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_de_death">
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_rf_death">
         <property name="text">
          <string>更新</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_feed">
     <attribute name="title">
      <string>喂食记录</string>
     </attribute>
     <widget class="QTableView" name="table_feed_time">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>741</width>
        <height>381</height>
       </rect>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QWidget" name="layoutWidget0">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>400</y>
        <width>721</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btn_cm_feed">
         <property name="text">
          <string>提交</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_rb_feed">
         <property name="text">
          <string>撤销</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_ad_feed">
         <property name="text">
          <string>添加</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_de_feed">
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_rf_feed">
         <property name="text">
          <string>更新</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_login">
     <attribute name="title">
      <string>登录记录表</string>
     </attribute>
     <widget class="QTableView" name="table_login">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>741</width>
        <height>381</height>
       </rect>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>400</y>
        <width>721</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="btn_cm_login">
         <property name="text">
          <string>提交</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_rb_login">
         <property name="text">
          <string>撤销</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_ad_login">
         <property name="text">
          <string>添加</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_de_login">
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_rf_login">
         <property name="text">
          <string>更新</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_user">
     <attribute name="title">
      <string>用户信息表</string>
     </attribute>
     <widget class="QTableView" name="table_user">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>741</width>
        <height>381</height>
       </rect>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>400</y>
        <width>721</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="btn_cm_user">
         <property name="text">
          <string>提交</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_rb_user">
         <property name="text">
          <string>撤销</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_ad_user">
         <property name="text">
          <string>添加</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_de_user">
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_rf_user">
         <property name="text">
          <string>更新</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>MOUSE MANAGE SYSTEM</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_welcome">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Welcome, NULL</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/image/logo1.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/image/mouse1.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="act_logoff"/>
    <addaction name="act_exit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="act_help"/>
    <addaction name="act_about"/>
    <addaction name="separator"/>
    <addaction name="act_Qt"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>刷新</string>
    </property>
    <addaction name="act_refresh"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="act_about">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="act_exit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="act_logoff">
   <property name="text">
    <string>注销</string>
   </property>
  </action>
  <action name="act_Qt">
   <property name="text">
    <string>Qt</string>
   </property>
  </action>
  <action name="act_help">
   <property name="text">
    <string>帮助</string>
   </property>
  </action>
  <action name="act_refresh">
   <property name="text">
    <string>刷新</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>btn_rf_mouse</tabstop>
  <tabstop>box_name</tabstop>
  <tabstop>btn_edit</tabstop>
  <tabstop>table_mouse_death</tabstop>
  <tabstop>table_feed_time</tabstop>
  <tabstop>table_user</tabstop>
  <tabstop>table_login</tabstop>
  <tabstop>table_mouse</tabstop>
  <tabstop>edit_name</tabstop>
  <tabstop>check_name</tabstop>
  <tabstop>tw_main</tabstop>
  <tabstop>birth_min</tabstop>
  <tabstop>check_birth</tabstop>
  <tabstop>check_lastmeal</tabstop>
  <tabstop>birth_max</tabstop>
  <tabstop>lastmeal_min</tabstop>
  <tabstop>lastmeal_max</tabstop>
  <tabstop>check_health</tabstop>
  <tabstop>box_health</tabstop>
  <tabstop>btn_search</tabstop>
  <tabstop>btn_reset</tabstop>
  <tabstop>check_master</tabstop>
  <tabstop>box_master</tabstop>
 </tabstops>
 <resources>
  <include location="logo.qrc"/>
 </resources>
 <connections/>
</ui>
